docs/pvecsictl.md

```diff
@@ -21,8 +21,8 @@ Available Commands:
 
 ### Migrate
 
-Migration requires the root privileges on the Proxmox cluster.
-You need to provide the cloud-config file with the root credentials (username/password) to the Proxmox cluster.
+Migration requires root privileges on the Proxmox cluster.
```

Comment:
@sergelogvinov I assume it's possible to create a role for this similar to the one for the `kubernetes-csi@pve` user?

Reply:
Unfortunately not... 
This is proxmox limitation. https://pve.proxmox.com/pve-docs/api-viewer/#/nodes/{node}/storage/{storage}/content/{volume} (POST method)

Reply:
Thanks for the clarification. I'll add that to the documentation.